pyproject.toml

```diff
@@ -72,6 +72,7 @@ dependencies = [
   "urllib3>=2.2.3",
   "xmltodict>=0.14.2",
   "openshift-python-wrapper~=4.18.0",
+  "marshmallow~=3.26.1",
```

Comment:
curious, why we don't have it in a main branch?

Reply:
it's added in main as well here https://github.com/RedHatQE/openshift-virtualization-tests/pull/815